src/app.rs

```diff
@@ -48,6 +48,11 @@ impl<T: Data + 'static> AppLauncher<T> {
         }
     }
 
+    pub fn use_logger(self) -> Self {
```

Comment:
This name is slightly misleading; there are many ways to use the a logger (for instance the user can set up logging manually, or in the future we might by default enable logging to the platform logging service) so what we're really doing here is printing logs to stderr. You could say `use_simple_logger` or `print_logs` or `print_log_messages` or something? And I'd also include a doc comment to explain that this is provided for debugging, during development.